exo-deploy/terraform/aws/network/network.tf

```diff
@@ -0,0 +1,28 @@
+module "vpc" {
+  source = "./vpc"
+
+  name = "${var.name}"
+  env  = "${var.env}"
+}
+
+module "subnets" {
+  source = "./subnets"
+
+  name               = "${var.name}"
+  env                = "${var.env}"
+  vpc_id             = "${module.vpc.id}"
+  vpc_cidr           = "${module.vpc.cidr}"
+  availability_zones = "${var.availability_zones}"
```

Comment:
+1 Good call, i don't think we would want to use more than 3. should probably set this in `aws.tf`